### PR TITLE
New package: font-tewi-2.0.0

### PR DESCRIPTION
**Pull request by asiekierka**

### CODE REVIEW

**Comment by Vaelatern** [comment]
Upstream calls it `tewi-font`, you will save yourself some pain if you just name the whole package `tewi-font`.

**Comment by asiekierka** [comment]
I feel that it's better to adhere to a standard naming, to make it easier to find the package.

File: srcpkgs/font-tewi/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'font-tewi'
+pkgname=font-tewi
+version=2.0.0
+revision=1
+wrksrc=tewi-font-${version}
+noarch="yes"
+makedepends="font-util bdftopcf python"
```
Comment (Vaelatern) [inline srcpkgs/font-tewi/template]: This variable should be: `hostmakedepends`

**Comment by Vaelatern** [comment]
I present you with `google-fonts-ttf` `ohsnap-font` `termsyn-font` `terminus-font` and xorg-fonts`. Just to name the ones that fit on my page. Yes, many are `font-such-and-such`, but they are (for the ones I've checked) named that way upstream.

**Comment by asiekierka** [comment]
I am aware, but the majority of the fonts begin with font- after all. I can change it if it's that big a problem...

**Comment by Vaelatern** [comment]
The fonts that are named that way, as far as I can tell, are either not named  with `font` in the name by upstream, or are named in exactly that way by upstream. Generally I think we go with what upstream sets. `xbps-query -Rs` can find it for us. The big deal is the `hostmakedepends` thing though.

**Comment by asiekierka** [comment]
Okay then, let's try again...

**Pull request closed without merging**